Multiprotocol/FX_nrf24l01.ino

```diff
@@ -144,6 +200,13 @@ static void __attribute__((unused)) FX_initialize_txid()
 		for(uint8_t i=1;i<FX_NUM_CHANNELS;i++)
 			hopping_frequency[i] = i*10 + hopping_frequency[0];
 	}
+	else // FX9630
+	{
+		#ifdef FORCE_FX9630_ID
+			memcpy(rx_tx_addr,(uint8_t*)"\xCE\x31\x9B\x73", 4);
+		#endif
+		memcpy(hopping_frequency,"\x13\x1A\x38", FX9630_NUM_CHANNELS);		//Original dump=19=0x13,26=0x1A,56=0x38
```

Comment:
Since freqs 1 and 2 are sent to the RX, I suggest to generate them instead of the hard coding.
Something like insert in line 205:
hopping_frequency[0] = 0x13;  // constant ??
hopping_frequency[1] = RX_num&0x0F + 0x1A;  // guessing
hopping_frequency[2] = rx_tx_addr[3]&0x0F + 0x38;  // guessing
move line 208 before #endif
Test that it works by changing the RX_num value and set hopping_frequency[0] to 0 to see if the RX uses the learnt freqs.

Reply:
I updated all your comments and now the code is working with non-fixed address.